src/components/search/results/next-queries-group.vue

```diff
@@ -1,11 +1,8 @@
 <template>
   <div class="x-list x-padding--06 x-background--neutral-95 x-list--gap-06">
     <div class="x-list x-list--gap-03">
-      <h1 class="x-title2 x-text--bold">{{ $t('nextQueriesGroup.title') }}</h1>
-      <p
-        v-html="$t('nextQueriesGroup.message', { query: $x.query.search })"
-        class="x-text x-font-size--05"
-      />
+      <h1 class="x-title3 x-text--bold">{{ $t('nextQueriesGroup.title') }}</h1>
+      <p v-html="$t('nextQueriesGroup.message', { query: $x.query.search })" class="x-text" />
```

Comment:
Be careful in the translations:

`  "message": "This is what other shoppers searched after <span class=\"x-text x-text--bold x-font-size--05\">\"{query}\"</span>"
` 

Remo the font-size--05 also in the nextQueriesGroup tranlations